.github/actions/github-script/fetch_keptn_versions.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Fetch latest release and prerelease
+prerelease=$(curl -s https://api.github.com/repos/keptn/keptn/releases | jq -r 'map(select(.prerelease)) | first | .tag_name')
+release=$(curl -s https://api.github.com/repos/keptn/keptn/releases | jq -r 'map(select(.prerelease==false)) | first | .tag_name')
```

Comment:
Since the releases page does include the releases in the order they are published (please correct me if I'm wrong), I don't think that the first release is every time the latest. Imagine the following scenario: 
- Keptn releases 0.18.0
- A bug is discovered in 0.16.1 so they release a fix with the version 0.16.2

Since the fix was releases after 0.18.0, it will probably be the first one in the list and I don't think that we want to test this version as latest ...

Reply:
After testing it a bit, it seems that the endpoint returns the items in the same order as on the Github releases page. We could use the `/latest` endpoint, which always gets the latest release of the repository, but the prerelease does not have such an endpoint. 

For fetching the prerelease, I believe the best solution would be to filter using the `tag_name`, which seems to work fine for the Keptn repository.